### ISSUE TITLE
A11y: Color Contrast. Blue on gray is not accessible

### ISSUE THREAD

**Issue by asangma** [Esri/calcite-app-components#568]
### Summary
When we use `$h-bb-060` on `--calcite-app-background-content` (`$blk-010`), the contrast does not pass WCAG AA contrast requirements.

### Actual Behavior
[Not enough contrast](https://contrast-ratio.com/#%23007ac2-on-%23f3f3f3).

### Expected Behavior
[Use $h-bb-070](https://contrast-ratio.com/#%2300619b-on-%23f3f3f3)

### Reproduction Steps

1.

### Working W3C Example/Tutorial 
https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0#qr-visual-audio-contrast-contrast

**Comment by asangma** [comment]
I believe the updates to the calcite-components values should handle this contrast issue. Closing.
